addons/medical_status/XEH_preInit.sqf

```diff
@@ -26,18 +26,30 @@ PREP_RECOMPILE_END;
 addMissionEventHandler ["EntityKilled", {_this call FUNC(handleKilledMission)}];
 
 if (hasInterface) then {
-    //Add Inventory action to uncon units
+    //Add inventory and open backpack actions to uncon units
     ["CAManBase", "init", {
         params ["_unit"];
 
         private _id = _unit addAction ["", {
-            params ["_target", "_caller", "_actionId", "_arguments"];
+            params ["_target", "_caller"];
 
             _caller action ["Gear", _target];
-        }, nil, 5.1, true, true, "gear", "_target getVariable ['ACE_isUnconscious',false] && {alive _target}", 3.5];
+        }, nil, 5.1, true, true, "gear", toString {
+            (_target isNotEqualTo ACE_player) &&
+            {!((lifeState _target) in ["HEALTHY", "INJURED"])}
+        }, 2];
 
         _unit setUserActionText [_id, localize "STR_ACTION_GEAR", "<img image='\A3\ui_f\data\igui\cfg\actions\gear_ca.paa' size='2.5' shadow=2 />"];
 
+        _unit addAction ["OpenBag", {
+            params ["_target", "_caller"];
+
+            _caller action ["OpenBag", _target];
```

Comment:
it does seem to work on dead people, but I also see this note
```
            // 26.8.2013 HOTFIX CardA: ATOpenBag can be processed with a live soldier's bag only
            // because some actions between bag and other containers(like ground,...)
            // needs unit->Brain() which is invalid for a dead soldier(dead soldier has no brain)
```